### PR TITLE
User profile & Views

### PR DESCRIPTION
**Pull request by BigiDigi89**
screenshot from 2019-02-09 18-57-04
screenshot from 2019-02-09 18-57-45
screenshot from 2019-02-09 18-58-19

### CODE REVIEW

File: app/uploaders/avatar_uploader.rb [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  #
+  # def scale(width, height)
+  #   # do something
+  # end
+
+  # Create different versions of your uploaded files:
+  version :thumb do
+    process resize_to_fit: [80, 80]
+  end
+
```
Comment (Gklmvch) [inline app/uploaders/avatar_uploader.rb]: Add larger img size for preview in user#show
```
version :portret do
  process resize_to_fit: [500, 500]
end
```

File: app/views/courses/_form.html.slim [hunk 2]
```diff
@@ -1,20 +1,19 @@
-.card.bg-light.mb-3.form-center
```
Comment (Gklmvch) [inline app/views/courses/_form.html.slim]: Why you delete form-center?
We use same style for all form views

File: app/views/devise/registrations/new.html.slim [hunk 3]
```diff
@@ -1,4 +1,4 @@
-.card.bg-light.mb-3.form-center
+.card.bg-light.mb-3
```
Comment (Gklmvch) [inline app/views/devise/registrations/new.html.slim]: Why you delete form-center?

File: app/views/devise/sessions/new.html.slim [hunk 4]
```diff
@@ -1,4 +1,4 @@
-.card.bg-light.mb-3.form-center
```
Comment (Gklmvch) [inline app/views/devise/sessions/new.html.slim]: Why you delete form-center?

File: app/views/users/_form.html.slim [hunk 5]
```diff
@@ -0,0 +1,17 @@
+.card.bg-light.mb-3
```
Comment (Gklmvch) [inline app/views/users/_form.html.slim]: Why you don't use form-center?

File: app/uploaders/avatar_uploader.rb (hunk 1 above)
Comment (BigiDigi89) [reply app/uploaders/avatar_uploader.rb]: You need to see avatar_for method in UsersHelper

**Pull request merged**